feature-utils/poly-import/test/polyIn-errors.test.js

```diff
@@ -0,0 +1,23 @@
+import { FileImportError, FileSelectionError, RefreshFilesError } from "../src";
+
+const classData = [
+    [FileImportError, "import"],
+    [FileSelectionError, "select"],
+    [RefreshFilesError, "refresh"],
+];
+
+describe("Errors have the right API ", () => {
+    it("throws correctly", () => {
+        classData.forEach(([testClass, testMsg]) => {
+            const testCause = "test";
```

Comment:
as this checks the `classData` (defined here and not in the real implementation), it seems to me it creates more tech-debt and potential issues if Error types are refactored at some point or more are added. 
We might want to fill this object dynamically from `PolyImportError` somehow and not on our own? (or any other alternative)

Reply:
Well, it checks that it's effectively closed for modification, as in the open/closed principle. I'll check anyway if it can be improved through encapsulation. And any way thanks for the suggestion, since this is still a draft.